app/transforms/dollar-cents.js

```diff
@@ -0,0 +1,13 @@
+import DS from 'ember-data';
+
+const { Transform } = DS;
+
+export default Transform.extend({
+  deserialize(serialized) {
+    return serialized  / 100;
+  },
+
+  serialize(deserialized) {
+    return Math.floor(deserialized * 100);
```

Comment:
Are there any potential issues we can run into with the math here? Potential rounding errors?